error.vue

```diff
@@ -1,76 +1,85 @@
 <template>
-  <div class="container">
-    <h1>Critical Failure!</h1>
-    <h2>Error {{ error?.statusCode }}</h2>
-    <p class="font-italics">{{ error?.message }}</p>
+  <div class="min-h-dvh text-darkness dark:bg-darkness dark:text-white">
+    <div class="container w-full sm:max-w-[40rem] sm:pl-56">
+      <div class="p-4">
```

Comment:
I changed the structure here slightly so that the error page content is positioned roughly how the main site content is positioned